src/ContainerInstance/help/New-AzContainerGroupVolumeObject.md

```diff
@@ -20,7 +20,7 @@ New-AzContainerGroupVolumeObject -Name <String> [-AzureFileReadOnly] [-AzureFile
 ```
 
 ## DESCRIPTION
-Create a in-memory object for Volume
+Create a in-memory object for Volume. Currently, we support mounting Azure File share as a volume or specifying git repository as volume directory. The empty directory volume and secret volume are not supported yet.
```

Comment:
I'm hesitating to do this job. The description is auto-generated, I'm afraid that this change will be overridden in next change. Is there any way to customize description of cmdlets? @dolauli

Reply:
The cmdlet is a custom cmdlet. So we could customize the description in https://github.com/CaptainFanZzz/azure-powershell/blob/update-New-AzContainerGroup/src/ContainerInstance/custom/New-AzContainerGroupVolumeObject.ps1#L19

Reply:
How about New-AzContainerGroup? It's a customized cmdlet after generated . Can we use the same way?

Reply:
> 

Yes.

Reply:
Thanks @dolauli .